app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
```

Comment:
Great job @erickhansx 🤝

- [ ] [Optional] I would suggest you add more styles to the logout and forgot password pages in order to improve the look of yoyr design.

Reply:
Will do so! Thank you so much my friend!